### DISCUSSION TITLE
disable specific payloads

### DISCUSSION THREAD

**Discussion by celesteking** [strongswan/strongswan#2166] [General]
In a road-warrior scenario I'd like to disable ikev2 DNS configuration payload pull on initiator side.
Pulling IPs is enabled by specifying `conn.vips`, but I haven't found a corresponding option for disabling/enabling resolvers payload.

Thanks.

**Comment by tobiasbrunner** [comment]
> In a road-warrior scenario I'd like to disable ikev2 DNS configuration payload pull on initiator side.

Why? What's the issue with that?

> but I haven't found a corresponding option for disabling/enabling resolvers payload.

If you disable plugins that handle DNS servers, no should be requested.  So make sure you don't have the _resolve_ plugin enabled.

**Comment by celesteking** [comment]
> Why? What's the issue with that?

Several connections on one host, for several users. Assuming there's no control over responder, I expected this to be configurable on the initiator side. But nevermind, I understand that this isn't possible to do per connection. Thanks.

**Reply by tobiasbrunner** [reply]
Not sure I understand. You have a host where multiple users concurrently initiate roadwarrior VPN connections? (Where potentially each tunnels all traffic to the remote or even a different remote.) And some would be allowed to change the DNS servers and some not?

**Comment by celesteking** [comment]
You're right, I have overcomplicated the setup. I should've gone with containers. With current setup, there's main road-warrior connection and then users are able to initiate their own, separate connections (to different responders) which "override" the main one. I didn't like the resolvers that were pulled, thus I wanted to keep our own (from main connection) in place. Nevermind, it's just my stupid ideas.
